Cognite.Simulator.Extensions/Types.cs

```diff
@@ -24,6 +26,11 @@ public class SimulatorModelInfo
     /// </summary>
     public class SimulatorRoutineRevisionInfo
     {
+        /// <summary>
+        /// The unique identifier of the routine revision.
+        /// </summary>
+        public long Id { get; set; }
```

Comment:
Should these properties be immutable, i.e. use `{ get; init; }`

Reply:
we are on .net 8, but I can make it `internal set` instead